redisinsight/ui/src/constants/commands.ts

```diff
@@ -62,6 +62,14 @@ export enum CommandGroup {
   AI = 'ai'
 }
 
+export enum CommandGroupsStartsWith {
```

Comment:
May be better name for this enum is `CommandPrefix`?

Reply:
yes, renamed

Reply:
yes, renamed